### PR TITLE
[new locale] oc: Occitan language

### PR DESCRIPTION
**Pull request by Quent-in**
In its lengadocian dialecte

### CODE REVIEW

**Comment by ichernev** [comment]
@Quent-in https://github.com/moment/moment/blob/develop/CONTRIBUTING.md#code-organization

You also need tests. Check the other locale files and tests in `src/locale/xx.js` and `src/test/locale/xx.js`.

**Comment by Quent-in** [comment]
I'm not good enough to do so I'm afraid :(

**Comment by ichernev** [comment]
@Quent-in you need to do a few things:
1. Name your locale `oc` and put it in `src/locale/oc.js`
1. Check another locale, like `src/locale/fr.js` or `src/locale/de.js`, and make sure yours looks similar (in terms of how you define it)
1. Put tests in `src/test/locale/oc.js` similar to `src/test/locale/en.js` (just copy paste the `en.js` and translate to Occitan

That's it!

File: locale/oc-lnc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        h : 'una ora',
+        hh : '%d oras',
+        d : 'un jorn',
+        dd : '%d jorns',
+        M : 'un mes',
+        MM : '%d meses',
+        y : 'un an',
+        yy : '%d ans'
+    },
+    ordinalParse: /\d{1,2}(r|n|t|è|a)/,
```
Comment (fbonzon) [inline locale/oc-lnc]: Please change to `dayOfMonthOrdinalParse`. This was recently updated in master branch.

**Comment by Quent-in** [comment]
I'll try to make the changes and run the tests as asked.

**Comment by marwahaha** [comment]
@Quent-in - let me know how it is going. You can email me at marwahaha@berkeley.edu if you are still having trouble.

**Comment by marwahaha** [comment]
@Quent-in - how is it going?

**Comment by marwahaha** [comment]
see https://github.com/moment/moment/pull/4624

**Pull request closed without merging**